modules/reference/pages/configuration/config-channels.adoc

```diff
@@ -1,3 +1,134 @@
+[[ref.webui.config.channels]]
 = Configuration Channels
 
+As mentioned above, {productname} manages both central and local configuration channels and files.
+Central configuration management allows you to deploy configuration files to multiple systems (both traditional clients ([Management]) and Salt minions ([Salt])).
+Local configuration management is available for traditional clients ([Management]) only and allows you to specify overrides or configuration files that are not changed by subscribing the system to a central channel.
+
+A "`state channel`" is a type of a configuration channel but for Salt minion only.
+For a state channel, the [path]``init.sls`` file is not auto-generated, the user creates and edits it.
+Additionally, state channels can contain arbitrary configuration files that could be referenced from within the [path]``init.sls`` file.
+Therefore, state channels effectively replace custom states.
+
+.Referencing Configuration Files with Organization ID
+[NOTE]
+====
+You must reference configuration files with the `salt://` prefix, the organization ID, and the channel name.
+For example, to reference [path]``/etc/motd`` use:
+
+----
+file.managed:  - source: salt://manager_org_1/`channel_name`/etc/motd
+----
+====
+
+
+Central configuration or state channels must be created via the links on this page.
+
+Click the name of the configuration channel to see the details page for that channel.
+If you click the number of files in the channel, you are taken to the [guimenu]``List/Remove Files`` page of that channel.
+If you click the number of systems subscribed to the configuration channel, you are taken to the menu:Configuration Channel Details[Systems > Subscribed   Systems] page for that channel.
```

Comment:
Remove the 'if you' statements, make it clear where these actions are taking place, and what the end goal is.